site/docs/pages/swap/types.mdx

```diff
@@ -139,7 +139,9 @@ type SwapQuote = {
 type SwapReact = {
   children: ReactNode;
   className?: string; // Optional className override for top div element.
-  config?: SwapConfig;
+  config?: {
+    maxSlippage: number; // Maximum acceptable slippage for a swap (e.g., 3 for 3%). This is a percentage, not basis points;
```

Comment:
Flagged from a user that we had no documentation of the maxSlippage param. SwapConfig is not an exported type and this page does not support twoslash